ios/RNPlayerViewManager.swift

```diff
@@ -100,6 +100,27 @@ class RNPlayerViewManager: RCTViewManager {
          }
      }
 
+     @objc func setScalingMode(_ viewId: NSNumber, scalingMode: String) {
+         bridge.uiManager.addUIBlock { [weak self] _, views in
+             guard
+                 let self,
+                 let view = views?[viewId] as? RNPlayerView
+             else {
+                 return
+             }
+             guard let playerView = view.playerView else {
+                 return
+             }
+             if scalingMode == "Zoom" {
+                 playerView.scalingMode = ScalingMode.zoom
+             } else if scalingMode == "Stretch" {
+                 playerView.scalingMode = ScalingMode.stretch
+             } else {
+                 playerView.scalingMode = ScalingMode.fit
+             }
```

Comment:
You could use `switch` for this to simplify this:
```suggestion
             switch scalingMode {
             case "Zoom":
                 playerView.scalingMode = .zoom
             case "Stretch":
                 playerView.scalingMode = .stretch
             case "Fit":
                 playerView.scalingMode = .fit
             default:
                 break
             }
```